src/discussions/post-comments/PostCommentsView.test.jsx

```diff
@@ -223,6 +223,34 @@ describe('ThreadView', () => {
       expect(JSON.parse(axiosMock.history.patch[axiosMock.history.patch.length - 1].data)).toMatchObject(data);
     }
 
+    it('should not show post footer', async () => {
+      Factory.resetAll();
+      await axiosMock.onGet(`${threadsApiUrl}${discussionPostId}/`)
+        .reply(200, Factory.build('thread', {
+          vote_count: 0,
+          following: false,
+          closed: false,
+          group_id: null,
+        }));
+      await executeThunk(fetchThread(discussionPostId), store.dispatch, store.getState);
+      renderComponent(discussionPostId);
```

Comment:
reuse the function getThreadAPIResponse to mock thread data and pass vote_count, following, closed and group_id as a parameter to reuse in multiple test cases